tests/nested/manual/refresh-revert-fundamentals/task.yaml

```
@@ -123,12 +119,6 @@ execute: |
     echo "Check the change is completed"
     case "$SNAP" in
         pc-kernel|core20)
-            for _ in $(seq 10); do
-                if tests.nested exec "snap changes" | MATCH "$REVERT_ID\s+Done\s+.*"; then
-                    break
-                fi
-                sleep 1
-            done
-            tests.nested exec "snap changes" | MATCH "$REVERT_ID\s+Done\s+.*"
+            tests.nested exec "retry --wait 1 -n 10 --env REVERT_ID=\"$REVERT_ID\" sh -c \'snap changes | MATCH \"$REVERT_ID\s+Done\s+.*\"\'"
```

now that we are using a binary for nested exec, could these be rewritten more simply as

```suggestion
            retry --wait 1 -n 10 --env REVERT_ID=\"$REVERT_ID\" sh -c "tests.nested exec snap changes | MATCH \"$REVERT_ID\s+Done\s+.*\""
```
Then we are only running snap changes in the nested env and running MATCH on the host. AFAICT that is slightly less quoting if I didn't mess it up

---

sure, done

---

I didn't see you change this particular one, but that's fine, let's handle those in followups to this so we can land this PR